tests/eval_files/async23x_asyncio.py

```diff
@@ -0,0 +1,46 @@
+# type: ignore
+# ARG --enable=ASYNC230,ASYNC231
+# NOTRIO # see async23x.py
+# NOANYIO # see async23x.py
+# BASE_LIBRARY asyncio
+import io
+import os
+
+import asyncio
+
+
+async def foo():
+    open("")  # ASYNC230_asyncio: 4, 'open'
+    io.open_code("")  # ASYNC230_asyncio: 4, 'io.open_code'
+    os.fdopen(0)  # ASYNC231_asyncio: 4, 'os.fdopen'
+
+    # sync call is awaited, so it can't be sync
+    await open("")
+
+    # asyncio.wrap_file does not exist
+    await asyncio.wrap_file(open(""))  # ASYNC230_asyncio: 28, 'open'
+    await asyncio.wrap_file(os.fdopen(0))  # ASYNC231_asyncio: 28, 'os.fdopen'
```

Comment:
Doesn't make sense to test this if it doesn't exist, IMO.

Reply:
The test is to make sure the protection that `wrap_file()` would give doesn't trigger. Not that it would matter much since it doesn't exist, but I constructed the file by copypasting and replacing - so /shrug. I'll clarify the comment